hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/storefiletracker/DefaultStoreFileTracker.java

```diff
@@ -33,7 +33,7 @@
 @InterfaceAudience.Private
 class DefaultStoreFileTracker extends StoreFileTrackerBase {
 
-  public DefaultStoreFileTracker(Configuration conf, TableName tableName, boolean isPrimaryReplica,
+  public DefaultStoreFileTracker(Configuration conf, TableName tableName, Boolean isPrimaryReplica,
```

Comment:
Why this change?

Reply:
It was giving me a nosuchmethod error with the ReflectionUtils method I was using. Switched to use ReflectionUtils.newInstance fixed this.